apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+(function (exports) {
+
+  var MoreAppsNavigation = {
+    controls: null,
+    selectedElemIndex: -1,
```

Comment:
May be it is more convenient to initialize selectedElemIndex there as zero value not as -1.